server/tests/integration/test_api_submission.py

```diff
@@ -106,23 +106,24 @@ def test_zip(self):
 			created='created',
 			get_messages=lambda: {'file_contents': {'gup.py': 1}},
 			assignment=self._assign.key,
-			to_json=lambda: {})
+			to_json=lambda: {},
+			can=lambda *args: True)
 		self.API().zip(obj, self.accounts['dummy_admin'], {})
-		
+
 	def test_zip_files(self):
 		""" Tests that zip_files does not crash """
 		name, file_contents = 'yolo', {'gup.py': 'import fish'}
 		name2, zipfile = self.API().zip_files(name, file_contents)
 		self.assertEqual(name, name2)
 		return name, zipfile
-		
+
 	def test_make_zip_response(self):
 		""" Check zipfile response headers """
 		with self.app.test_request_context('/api/v2'):
```

Comment:
Not sure. I copied this from another section of the code. Do you think it's supposed to be v1?